app/app/__init__.py

```diff
@@ -0,0 +1,41 @@
+from flask import Flask
+from flask_sqlalchemy import SQLAlchemy
+from os import path
+from flask_login import LoginManager
```

Comment:
not sure if we want to enforce this coding discipline but... at work, we organise are imports based on where they come from

eg.

```
from os import path

from flask import Flask
from flask_sqlalchemy import SQLAlchemy
from flask_login import LoginManager

from app.model import User # THIS IS JUST AN EXAMPLE
```

in these scenario, there are three types of imports:
1. Python-specific libraries / other libraries not related to the main framework
2. Flask-specific libraries
3. project-specific modules (if you are importing from the same codebase)

I won't enforce this as much... simply becausse this is a uni project, but good to know/be aware of as it increases code readability among developers